neasqc_wp61/models/quantum/embedder.py

```diff
@@ -98,13 +152,12 @@ def save_embedding_dataset(self, path: str, filename: str) -> None:
             If the method has not been implemented by subclasses.
         """
         try:
-            self.vectorised_sentences = self.dataset[
-                "sentence_vectorised"
-            ].tolist()
+            _ = self.dataset["sentence_vectorised"].tolist()
         except KeyError:
```

Comment:
Afaik you cannot raise 2 errors at once.
Also, even if you could you definitely don't want to do so. Pick one and stick to it.